src/OpenApiService.php

```diff
@@ -148,6 +155,17 @@ public function createSchema(): OpenApi
 		return $openApi;
 	}
 
+	public function setServers(array $servers): void
+	{
+		foreach ($servers AS $server) {
```

Comment:
Lowercase please

Reply:
Sorry, I'm used to write uppercase this one => fixed ;)